### ISSUE TITLE
Accessibility: Set tabindex property on `ng-clear-wrapper`

### ISSUE THREAD

**Issue by krunal-jethva-tark** [ng-select/ng-select#2438]
### Is your feature request related to a problem? Please describe

My requirements for this is very simple I would like to set `tabindex` attribute on clear button to `-1` so it does not get included in tab order and end user does not have to press `tab` twice to focus out `ng-select` element.

### Describe the solution you'd like

I would want to a `@Input` attribute in ng-select to set this behavior and if possible, configure this behavior at root level so I don't have to configure it at all instance of `ng-select`

### Describe alternatives you've considered

```typescript
import { Directive, ElementRef, AfterViewInit } from '@angular/core';

@Directive({
  selector: '[disableClearTab]'
})
export class DisableClearTabDirective implements AfterViewInit {

  constructor(private el: ElementRef) { }

  ngAfterViewInit() {
    const clearButton = this.el.nativeElement.querySelector('.ng-clear-wrapper');
    if (clearButton) {
      clearButton.setAttribute('tabindex', '-1');
    }
  }
}
```

I have considered to write a directive something like this and then put it in an every `ng-select` element. but i suppose there should be a default behavior by `ng-select` element which allows to set `tabindex` or disable `tabindex` on clear button.

**Comment by JAVA-MSDT** [comment]
> [@pavankjadda](https://github.com/pavankjadda) any news :)?

Can you check this PR, maybe it has your solution, if it is please reopened and ask them to review 
https://github.com/ng-select/ng-select/pull/2022
